src/routes/safe/components/Transactions/TxList/TxOwners.tsx

```diff
@@ -1,18 +1,94 @@
-import { Text, Icon } from '@gnosis.pm/safe-react-components'
-import { ReactElement } from 'react'
-import styled from 'styled-components'
-
-import Img from 'src/components/layout/Img'
-import { ExpandedTxDetails, isModuleExecutionInfo } from 'src/logic/safe/store/models/types/gateway.d'
-import TransactionListActive from './assets/transactions-list-active.svg'
-import TransactionListInactive from './assets/transactions-list-inactive.svg'
-import { AddressInfo } from './AddressInfo'
-import { OwnerList, OwnerListItem } from './styled'
-import { isCancelTxDetails } from './utils'
-
-const StyledImg = styled(Img)`
-  background-color: transparent;
-  border-radius: 50%;
+import { ReactElement, useState } from 'react'
+import styled, { AnyStyledComponent } from 'styled-components'
+import Step from '@material-ui/core/Step'
+import StepConnector from '@material-ui/core/StepConnector'
+import StepContent from '@material-ui/core/StepContent'
+import StepLabel from '@material-ui/core/StepLabel'
+import Stepper from '@material-ui/core/Stepper'
+import FiberManualRecordIcon from '@material-ui/icons/FiberManualRecord'
+import AddCircleIcon from '@material-ui/icons/AddCircle'
+import RadioButtonUncheckedOutlinedIcon from '@material-ui/icons/RadioButtonUncheckedOutlined'
+import CheckCircleIcon from '@material-ui/icons/CheckCircle'
+import CancelIcon from '@material-ui/icons/Cancel'
+
+import { ExpandedTxDetails, isMultiSigExecutionDetails } from 'src/logic/safe/store/models/types/gateway.d'
+import { AddressInfo } from 'src/routes/safe/components/Transactions/TxList/AddressInfo'
+import { isCancelTxDetails } from 'src/routes/safe/components/Transactions/TxList/utils'
+import { black300, gray500, primary400, red400 } from 'src/theme/variables'
+
+// Icons
+
+// All icons from MUI share the same type
+const getStyledIcon = (icon: typeof AddCircleIcon): AnyStyledComponent => {
+  return styled(icon)`
+    height: 20px;
+    width: 20px;
+    margin-left: 2px;
+  `
+}
+
+const TxCreationIcon = getStyledIcon(AddCircleIcon)
+const TxRejectionIcon = getStyledIcon(CancelIcon)
+const CheckIcon = getStyledIcon(CheckCircleIcon)
+
+const CircleIcon = styled(getStyledIcon(RadioButtonUncheckedOutlinedIcon))`
+  stroke: currentColor;
+  stroke-width: 1px;
+`
+const DotIcon = styled(FiberManualRecordIcon)`
+  height: 14px;
+  width: 14px;
+  margin-left: 5px;
+`
+
+// Stepper
+const StyledStepper = styled(Stepper)`
+  padding: 0;
+`
+
+const StyledStepConnector = styled(StepConnector)`
+  padding: 3px 0;
+
+  .MuiStepConnector-line {
+    margin-left: -1px;
+    border-color: ${gray500};
+    border-left-width: 2px;
+    min-height: 14px;
+  }
+`
+
+type StepColor = 'green' | 'gray' | 'orange' | 'red'
+const getStepColor = (color: StepColor): string => {
+  switch (color) {
+    case 'green':
```

Comment:
We shouldn't name colors by their hue, but rather use semantic names like "confirmed", "inactive", "error" etc.

Reply:
I chose this because there are some default ones built into the stepper: `confirmed`, `active` and `error` but they do not align with how we want to show them. The 'Confirmations' step is orange and then the individual confirmations that follow are green. What do you think?